src/xdp/program.h

```diff
@@ -10,15 +10,28 @@
 typedef struct _XDP_PROGRAM XDP_PROGRAM;
 typedef struct _XDP_RX_QUEUE XDP_RX_QUEUE;
 
+//
+// TODO: eBPF should publish opaque structure.
```

Comment:
Issue?

Reply:
Yup, #170 and https://github.com/microsoft/ebpf-for-windows/issues/2146